lib/Conversion/XTenNNToTorch.cpp

```diff
@@ -154,7 +158,39 @@ ValueRange groupConv2dToTorch(GroupConv2dOp op, GroupConv2dOp::Adaptor adaptor,
       ->getResults();
 }
 
-ValueRange padReflectToTorch(ReflectPadOp op, ReflectPadOp::Adaptor adaptor,
+std::optional<ValueRange> resizeToTorch(ResizeOp op, ResizeOp::Adaptor adaptor,
+                        ArrayRef<Type> types, ValueRange values,
+                        ConversionPatternRewriter &rewriter) {
+  auto loc = op->getLoc();
+  auto opName = rewriter.getStringAttr("onnx.Resize");
+  // Mode Nearest is supported via conversion to tosa and mostly unsupported by torch.operator "onnx" 
+  if (adaptor.getMode() == 0)
+    return std::nullopt;
+  std::string modeStr = "linear";
+  llvm::SmallVector<std::string, 4> numberToTransMode = {"half_pixel", "pytorch_half_pixel", "asymmetric", "align_corners"};
+  std::string coordinateTransStr = numberToTransMode[adaptor.getCoordinateTransformationMode()];
```

Comment:
This looks potentially dangerous, e.g. when new modes get added. Can we check here that the access is not out of bounds?